src/index.html

```diff
@@ -0,0 +1,1302 @@
+          <article class="flex-box">
+            <figure>
+              <div class="article-img flex">
+                <img src="../src/assets/weed.png" alt="picture of a cannabis leaf">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  Scientists modified a strain of cannabis to kill cancer cells
+                </a>
+                Tristan Greene - 12 hours ago
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box">
+            <figure>
+              <div class="article-img flex">
+                <img src="../src/assets/game-boy.png" alt="picture of a future game-boy console">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  Analogue's retro handheld Pocket won't ship until 2021
+                </a>
+                Rachel Kasar - 12 hours ago
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box">
+            <figure>
+              <div class="article-img flex">
+                <img src="../src/assets/covid.png" alt="picture of coronavirus logo">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  Blame the neanderthals
+                </a>
+                Tristan Greene - 15 hours ago
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box">
+            <figure>
+              <div class="article-img flex">
+                <img src="../src/assets/trump.png" alt="picture of trump">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  This AI turns your home videos into cute cartoons
+                </a>
+                Thomas - 15 hours ago
+              </figcaption>
+            </figure>
+          </article>
+          <article class="flex-box">
+            <figure>
+              <div class="article-img flex">
+                <img src="../src/assets/meds.png" alt="picture of medication">
+              </div>
+              <figcaption class="flex">
+                <a href="#">
+                  Kodak, now a pharma company, expects drugs to make up 30-40%
+                </a>
+                David Canellis - 17 hours ago
+              </figcaption>
+            </figure>
+          </article>
+        </div>
+      </section>
+
+      <!-- END of the latest news section -->
+
+      <!-- START of funding section -->
+      <section class="funding-container">
+        <div class="funding-titles flex">
+          <h2 class="fund-left flex">
+            <a href="#">
+              Latest funding rounds on Index.co >
+            </a>
+          </h2>
+
+          <ul class="fund-right flex">
+            <li>
+              <a href="#">
+                Add Info
+              </a>
+            </li>
```

Comment:
- [ ] Same problem here with the breakpoints.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/89059126-0217d600-d337-11ea-8782-cfc451a9f69a.png)|![image](https://user-images.githubusercontent.com/61479648/89059150-0a701100-d337-11ea-9e93-c4b83dcd356a.png)  |